### DISCUSSION TITLE
kpanda/user-guide/configmaps-secrets/create-secret

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2317] [Announcements]
# kpanda/user-guide/configmaps-secrets/create-secret

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/configmaps-secrets/create-secret.html

**Comment by liyuerich** [comment]
能给 自定义 类型 一个详细的说明吗？
